runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/client/grpc/GrpcDirectGetWorkStream.java

```diff
@@ -80,7 +89,7 @@ final class GrpcDirectGetWorkStream
    */
   private final ConcurrentMap<Long, GetWorkResponseChunkAssembler> workItemAssemblers;
 
-  private final boolean multipleItemsInGetWorkResponse;
+  private final boolean batchedGetWorkResponse;
```

Comment:
this too, I think it is easier to understand what it is. otherwise the name sounds like it is a response to me

ditto for the other files